arches/app/utils/data_management/resources/importer.py

```diff
@@ -123,42 +125,53 @@ def __init__(self, file=None, mapping_file=None, relations_file=None, relation_c
+                if mapping != None:
+                    reader = CsvReader()
+                    reader.import_business_data(business_data=business_data, mapping=mapping, overwrite=overwrite, bulk=bulk)
+                else:
+                    print '*'*80
+                    print 'ERROR: No mapping file detected. Please indicate one with the \'-c\' paramater or place one in the same directory as your business data.'
+                    print '*'*80
+                    sys.exit()
+            elif file_format == 'shp':
+                # if mapping != None:
+                #     SHPFileImporter().import_business_data(business_data, mapping)
+                # else:
+                #     print '*'*80
+                #     print 'ERROR: No mapping file detected. Please indicate one with the \'-c\' paramater or place one in the same directory as your business data.'
+                #     print '*'*80
+                #     sys.exit()
+                pass
 
-        if file_format == None:
-            file_format = self.file_format
-        if business_data == None:
-            business_data = self.business_data
-        if mapping == None:
-            mapping = self.mapping
-        if file_format == 'json':
-            reader = ArchesFileReader()
-            reader.import_business_data(business_data, mapping)
-        elif file_format == 'csv':
-            if mapping != None:
-                reader = CsvReader()
-                reader.import_business_data(business_data=business_data, mapping=mapping, overwrite=overwrite, bulk=bulk)
-            else:
-                print '*'*80
-                print 'ERROR: No mapping file detected. Please indicate one with the \'-c\' paramater or place one in the same directory as your business data.'
-                print '*'*80
-                sys.exit()
-        elif file_format == 'shp':
-            # if mapping != None:
-            #     SHPFileImporter().import_business_data(business_data, mapping)
-            # else:
-            #     print '*'*80
-            #     print 'ERROR: No mapping file detected. Please indicate one with the \'-c\' paramater or place one in the same directory as your business data.'
-            #     print '*'*80
-            #     sys.exit()
-            pass
-
-        # Import resource to resource relationships
-        reader.import_relations(relation_configs=self.relation_configs, relations=self.relations)
+            # Import resource to resource relationships
+            reader.import_relations(relation_configs=self.relation_configs, relations=self.relations)
 
-        elapsed = (time() - start)
-        print 'Time to import_business_data = {0}'.format(datetime.timedelta(seconds=elapsed))
+            elapsed = (time() - start)
+            print 'Time to import_business_data = {0}'.format(datetime.timedelta(seconds=elapsed))
 
-        reader.report_errors()
+            reader.report_errors()
+        except:
```

Comment:
I believe you can remove this except block and just have try/finally (i'm pretty sure this is true).  With the except clause empty you essentially swallow all exceptions and if an error happens nothing will be reported to the user.   Bugs can then be very hard to trace.